lbrynet/core/client/ConnectionManager.py

```diff
@@ -19,6 +19,8 @@ def __init__(self, request_creators, factory):
 
 class ConnectionManager(object):
     implements(interfaces.IConnectionManager)
+    callLater = reactor.callLater
```

Comment:
I think that's good , in tests you need to be able to set callLater for all classes that uses it, and it would be better if you could just set a single function.  
I think this would be a seprate pull request, I see a lot callLaters in lbrynet that needs to be changed.